youtube.py

```diff
@@ -42,12 +42,15 @@ def get_service_result(self, url) -> ServiceResult:
             return ServiceResult(False, 0, 0, '')
 
         query = parse.parse_qs(parse.urlsplit(final_url.geturl()).query)
+        try:
+            request = self.youtubeClient.videos().list(
+                part="statistics",
+                id=query['v'][0]
+            )
+            response = request.execute()
+            view_count = int(response['items'][0]['statistics']['viewCount'])
+
+            return ServiceResult(view_count > self.threshold, view_count, self.threshold, 'YouTube plays')
```

Comment:
maybe add the youtube link here so we can see what the reason for crashing was

Reply:
I actually originally had it print that to console but then removed it because I was worried it fell into the realm of #17.  Happy to readd if we want it as a stop gap until #17 is knocked out.

Reply:
@martijnboers : Opted to go ahead and print it on that specific error and let the other exceptions pass rather than have a straight up "Exception" catch.